Sources/Web3Wallet/Web3Wallet.swift

```diff
@@ -0,0 +1,51 @@
+import Foundation
+import Combine
+
+import Auth
+import WalletConnectSign
```

Comment:
please check how we are doing @_exported imports in other packages. Otherwise it will break cocoapods releases

Reply:
just checked, this also need to do for Push SDK @llbartekll